README.md

```diff
@@ -1,14 +1,13 @@
 [![Build Status](https://travis-ci.org/eclipse-ee4j/jsonp.svg?branch=master)](https://travis-ci.org/eclipse-ee4j/jsonp)
 
-# JSON Processing (JSON-P)
+# Jakarta JSON Processing (JSON-P)
 
-This projects contains API and Reference Implementation (RI) of JSR-374. 
-The Java API for JSON Processing provides portable APIs to parse, generate, transform, and query JSON documents.
+The Java API for JSON Processing provides portable APIs to parse, generate, transform, and query JSON documents. This projects contains Jakarta JSON Processing specification, API and a compatible implementation.
```

Comment:
either 'this project' or 'these projects' but not 'this projects'

Reply:
ooops...

Reply:
Fixed.